orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/AdminController.groovy

```diff
@@ -57,6 +66,15 @@ class AdminController {
     noDiscoveryApplicationStatusPublisher.setInstanceEnabled(enabled)
   }
 
+  @RequestMapping(value = "/queue/redrive", method = RequestMethod.POST)
```

Comment:
/queue/dlq/redrive? This PR is a good start for additional /queue/dlq/* commands.